### ISSUE TITLE
Second image only visible, if followed by text

### ISSUE THREAD

**Issue by xundeenergie** [friendica/friendica#12887]
- [ ] I have searched open and closed issues for duplicates

### Bug Description
Images are only shown, if followed by some text. 
Urlpreview of image-links are also created.

### Steps to Reproduce
I created a posting with one image.
The image is shown.
Then i edited this posting and added a second image.
The first image is shown, the second image not... it is a url-preview in this case

![grafik](https://user-images.githubusercontent.com/5894829/225008743-49296620-f856-416c-9561-a7fcfea0b3c0.png)

Then i edited this posting again and added a few characters after the second image (because the url is there!!)
and get this

![grafik](https://user-images.githubusercontent.com/5894829/225009079-d91767cb-06d4-46ae-ace7-c955bcc2896d.png)
 (Yes, the same image twice!!)
 
 The image is shown, and also an urlpreview of the second image below.

Actual Result:
Image is shown only if followed by characters, Urlpreview of imagelink is created.

Expected Result:

Images should just be shown as images... all of them, nothing more.

### Screenshots

look above

### Platform Info

Friendica Version: 2023.03-rc

Friendica Source: git

PHP version: 8.1

SQL version: mariadb 10.5.18

**Comment by tobiasd** [comment]
Can you provide a link to a not-working posting?

I'm on 14647acce25719c3b787773d1367493b437b6342 (#12889) and I do see postings with more then one images attached at least. On a quick search I only found one of my own postings with "text - image - text - image" that was displayed correctly.

**Comment by xundeenergie** [comment]
Now i'm on my smartphone. Can not reproduce it here.

Maybe this is a case of displaysize?

Whrn i'm back on my laptop, i will try to reproduce it.

**Comment by annando** [comment]
You said the magic word: "Edit". So the problem possibly lies within the editing.

**Comment by xundeenergie** [comment]
> You said the magic word: "Edit". So the problem possibly lies within the editing.

I had it on fresh postings too..


And i get a urlpreview for own Images...

**Comment by xundeenergie** [comment]
This is strange... can not reproduce it.

**Comment by xundeenergie** [comment]
Oh... i found a posting, which ist weird.

https://soc.schuerz.at/display/4edd2508-1164-106f-682d-9c2732654022

There is a link-Preview. I can not delete. It is recreated.

This is one of this interesting posting from yesterday, where i created this issue.
The text is only:

```
[url=https://soc.schuerz.at/photos/jakob/image/152623972164106cf49b0b1846862579][img=https://soc.schuerz.at/photo/152623972164106cf49b0b1846862579-0.jpg][/img][/url]
[url=https://soc.schuerz.at/photos/jakob/image/152623972164106cf49b0b1846862579][img=https://soc.schuerz.at/photo/152623972164106cf49b0b1846862579-0.jpg][/img][/url]
blafoo
```

Why is there a linkPreview generated?

**Comment by xundeenergie** [comment]
I created a new posting https://soc.schuerz.at/display/4edd2508-1964-122d-4a30-171634075341

And i copied the url of the image from the last comment.
Pasted it in twice.

The sourcecode of this posting is now:

```
[url=https://soc.schuerz.at/photos/jakob/image/152623972164106cf49b0b1846862579][img=https://soc.schuerz.at/photo/152623972164106cf49b0b1846862579-0.jpg][/img][/url]
[url=https://soc.schuerz.at/photos/jakob/image/152623972164106cf49b0b1846862579][img=https://soc.schuerz.at/photo/152623972164106cf49b0b1846862579-0.jpg][/img][/url]
```

and it looks this:
![grafik](https://user-images.githubusercontent.com/5894829/225437210-ff867cf3-2769-4b8f-9109-36045377023e.png)

This is freshly created. Not edited.
i just added the "y" at the end... look, the second image appears.

```
[url=https://soc.schuerz.at/photos/jakob/image/152623972164106cf49b0b1846862579][img=https://soc.schuerz.at/photo/152623972164106cf49b0b1846862579-0.jpg][/img][/url]
[url=https://soc.schuerz.at/photos/jakob/image/152623972164106cf49b0b1846862579][img=https://soc.schuerz.at/photo/152623972164106cf49b0b1846862579-0.jpg][/img][/url]
y
```

![grafik](https://user-images.githubusercontent.com/5894829/225437393-a5dcc93c-3c60-4c35-9cc2-298da3343b46.png)

And when i remove the "y", the second image disappears...

**Comment by xundeenergie** [comment]
Please reopen this issue. @MrPetovan

**Comment by annando** [comment]
I guess I know the reason for that. And the PR for that will be done tomorrow after some more tests.

**Comment by xundeenergie** [comment]
Great! 
Thank you!!

**Comment by xundeenergie** [comment]
Maybe it helps...

The browser-console gives me this warning, when i load my site:
![grafik](https://user-images.githubusercontent.com/5894829/225460755-10460313-f5f2-4d3b-ad1f-4b8665fe3d6e.png)

404 - not found

and the urls are exactly the missing one from the critical postings from above.

**Comment by davidspring** [comment]
I have worked with many online editing tools that have this same problem. I think one option is to provide a Help tip in the editor to let posters know that there should always be at least one sentence after every image.  It is also helpful to have at least one sentence between each image.

**Comment by annando** [comment]
@davidspring no, there should be no need to enter text between the pictures. I also don't know if the issue still exists and also if all the issues described here are really of the same origin. I know however that we fixed some stuff connected to the pictures in the upcoming release.
